otter/test/test_auth.py

```diff
@@ -827,12 +856,11 @@ def test_composition(self):
         self.assertEqual(ra._max_retries, 3)
         self.assertEqual(ra._retry_interval, 5)
 
-        ia = ra._authenticator
-        self.assertIsInstance(ia, ImpersonatingAuthenticator)
-        self.assertEqual(ia._identity_admin_user, 'uname')
-        self.assertEqual(ia._identity_admin_password, 'pwd')
-        self.assertEqual(ia._url, 'htp')
-        self.assertEqual(ia._admin_url, 'ad')
+        st = ra._authenticator
+        self.assertIsInstance(st, SingleTenantAuthenticator)
```

Comment:
Nope; we're not changing the default authenticator. We're giving people who have regular user credentials (as opposed to admin credentials) the option to run otter, too.

There should be a test that checks composition of the single tenant authenticator, yes, but the existing impersonating authenticator tests shouldn't go away, and the single-tenant authenticator shouldn't be the default.